site/src/Librecores/ProjectRepoBundle/Resources/views/Project/settings.html.twig

```diff
@@ -129,7 +151,11 @@
       <h3 class="panel-title">Delete</h3>
     </div>
     <div class="panel-body">
-<a href="{{ path('librecores_project_repo_project_delete', {'parentName':project.getParentName(), 'projectName': project.getName()}) }}" class="btn btn-default">Delete this project</a>
+      <form id="myForm" class="delete-panel-body">
+	Enter the project name as displayed: <br />
+	<input type="text" id="delete-project-name" name="project_name" /><br /><br>
+	<input type="button" id="delete-project-button" disabled="disabled" onclick="window.location.href='{{ path('librecores_project_repo_project_delete', {'parentName':project.getParentName(), 'projectName': project.getName()}) }}'" class="btn btn-default" value="Delete this project"/>
```

Comment:
No inline JS. We don't have a CSP set right now but, in case we do, it's going to hurt us.

Reply:
You mean the onclick attribute here ?

Reply:
I moved this inline event handler to the top of the file.